pkg/driver/controller_server.go

```diff
@@ -224,7 +224,7 @@ func (driver *Driver) CreateVolume(ctx context.Context, request *csi.CreateVolum
 	createParameters, err := driver.flavor.ConfigureAnnotations(request.Name, request.Parameters)
 	if err != nil {
 		log.Errorf("Failed to configure create parameters from PVC annotations. err=%v", err)
-		return nil, status.Error(codes.Internal, "Failed to configure create parameters from PVC annotations")
+		return nil, status.Error(codes.Internal, fmt.Sprintf("Failed to configure create parameters from PVC annotations, %s", err.Error()))
```

Comment:
I haven't seen a comma used as a message separator like this.  A colon is often used, and sometimes a period.  I suggest choosing between those two.

Reply:
We have it at many places in the file, but sure will update.